.github/ISSUE_TEMPLATE/NEW_SYNTAX.md

```diff
@@ -32,12 +32,12 @@ assignees: ''
 
 Please check off all items that have already been completed. Be sure to paste the pull request URLs next to each item so we can verify the work as done.
 
-- [ ] Ecma262 update: <!-- paste PR URL for this syntax here -->
```

Comment:
I’m pretty sure this needs to be a dash in order to create a checklist in the issue. This isn’t just a regular bullet list.

Reply:
Looking at the GitHub docs for checklists I think you're right. But that's going to conflict with the changes I made in `.markdownlint.yml` as `* ` is now the required way, so there are a couple of ways I could "fix" this:

1. Revert the changes in `.markdownlint.yml `. This would reinstate the old behaviour of consistency within single files, but not across the repo. So this PR brings them all in line, but they could drift apart again in the future.
1. Add `NEW_SYNTAX.md` and `PULL_REQUEST_TEMPLATE.md` to `.markdownlintignore`. But this would also suppress other (future) linting issues in those files.
1. Add `<!-- markdownlint-disable-file MD004 -->` to the top/bottom of those two files. This would probably show up in the GitHub UI when creating a PR though.
1. Wrap those specific lines with `<!-- markdownlint-disable MD004 -->` and `<!-- markdownlint-enable MD005 -->` in those two files. This would probably show up in the GitHub UI when creating a PR though.

All of them have some negative side-effects, so @nzakas which one do you prefer that I implement?

Reply:
I think option 3 makes the most sense for now. We should really convert this file into YAML format anyway.